api/src/routes/edit.ts

```diff
@@ -2,9 +2,32 @@ import express = require("express");
 import Config from "../models/Config";
 const router = express.Router();
 import { requireToken } from "./auth";
+import { pidSanitizer } from "./sanitize";
+import Solr from "../services/Solr";
 
 router.get("/models", requireToken, function (req, res) {
     res.json({ CollectionModels: Config.getInstance().collectionModels, DataModels: Config.getInstance().dataModels });
 });
 
+async function getChildren(req, res) {
+    const query =
+        (req.params.pid ?? "").length > 0
+            ? 'fedora_parent_id_str_mv:"' + req.params.pid.replace('"', "") + '"'
```

Comment:
you can `"${variable}"`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Good point -- improved in 598aea2.